wikimedia/wikipedia-ios
# Pop up log in for logged out users

**Phabricator:** 
https://phabricator.wikimedia.org/T379794

### Notes
* 

### Test Steps
1. Fresh install, logged out, 6+ in history, and region/language 
2. Navigate to profile, then Year in Review
3. Make note of the pop up, select no thanks
4. Ensure you navigate to the YIR after no thanks
5. Repeat again but this time log in and ensure you can navigate to YIR


### Screenshots/Videos

https://github.com/user-attachments/assets/403aec21-05a0-4259-88ca-71acfcd0bd28
